test/trivial/deitz/statements/multi_decl4.chpl

```diff
@@ -1,7 +1,19 @@
-var x, y: sync int;
+//var x, y: sync int;
 
-x = 1;
-y = 2;
+var x: sync int;
+var y: x.type;
+
+/* Works:
+     var x: sync int;
+     var y: sync int;
+*/
+
+// Works:
+//   var x: sync int, y: sync int;
+
+
+x.writeEF(1);
+y.writeEF(2);
```

Comment:
It looks to me like we could revert the changes to this test.  I suspect my original rewrite was to keep it working at some point that it wasn't, and am not seeing any reason for the change anymore?

Reply:
I've reverted it, thanks.